pkgs/applications/kde/kmousetool.nix

```diff
@@ -0,0 +1,35 @@
+{ mkDerivation
+, lib
+, extra-cmake-modules
+, kdoctools
+, ki18n
+, kiconthemes
+, knotifications
+, kxmlgui
+, kwindowsystem
+, phonon
+, libXtst
+, libXt
+}:
+
+mkDerivation {
+  pname = "kmousetool";
+  meta = {
+    homepage = "https://github.com/KDE/kmousetool";
+    description = "Program that clicks the mouse for you";
+    license = with lib.licenses; [ gpl2 fdl12 ];
+    maintainers = [ lib.maintainers.jayesh-bhoot ];
+  };
```

Comment:
Ok. I simply copied from https://raw.githubusercontent.com/NixOS/nixpkgs/master/pkgs/applications/kde/gwenview.nix. It seems convention must have changed since then.

The latest commit moves `meta` to the bottom.